### PR TITLE
Age demographic data table displays unknown ages.

### PR DESCRIPTION
**Pull request by dsjen**
- added natural sort (so "unknown" would be sorted properly in conjunction with numbers-a.k.a. ages).
- updated age percent calculation to include unknowns
- added tests for data-table-view.js

@rocha @clintonb

### CODE REVIEW

**Comment by clintonb** [comment]
@dsjen Would you add a screenshot, please?

File: analytics_dashboard/static/js/utils/utils.js [hunk 1]
```diff
@@ -67,6 +67,64 @@ define(['moment', 'underscore', 'utils/globalization'], function (moment, _, Glo
...
+        },
+
+        /**
+         * Natural sort for ordering numbers and strings in the same list.
+         *
+         * From:
+         *  - http://www.overset.com/2008/09/01/javascript-natural-sort-algorithm-with-unicode-support/
+         *  - http://js-naturalsort.googlecode.com/svn/trunk/naturalSort.js
+         */
+        naturalSort: function (a, b) {
```
Comment (clintonb) [inline analytics_dashboard/static/js/utils/utils.js]: Download naturalSort.js from https://github.com/overset/javascript-natural-sort and place it in the vendor directory.

**Comment by dsjen** [comment]
Example age table sorted by age:
age-table

File: analytics_dashboard/static/js/utils/utils.js (hunk 1 above)
Comment (dsjen) [inline analytics_dashboard/static/js/utils/utils.js]: Good idea!  Added package to bower and loaded via require.

**Comment by clintonb** [comment]
:+1:

**Pull request merged**